src/spac/data_utils.py

```diff
@@ -346,37 +347,43 @@ def load_csv_files(file_names):
                             "Please check that the file is a valid CSV."
             raise TypeError(error_message)
 
-        current_schema = current_df.columns.to_list()
+        current_df["loaded_file_name"] = file_name
 
-        if len(meta_schema) == 0:
-            meta_schema = current_schema
-            print("Meta schema acquired. Columns are:")
-            for column_name in meta_schema:
-                print(column_name)
+        # current_schema = current_df.columns.to_list()
 
-        if len(meta_schema) == len(current_schema):
-            if set(meta_schema) != set(current_schema):
-                error_message = "Column in current file does not match " + \
-                        f"the meta_schema, got:\n {current_schema}. "
-                raise ValueError(error_message)
-        else:
-            error_message = "Column in current file does not match " + \
-                        f"the meta_schema, got:\n {current_schema}. "
-            raise ValueError(error_message)
+        # if len(meta_schema) == 0:
+        #     meta_schema = current_schema
+        #     print("Meta schema acquired. Columns are:")
+        #     for column_name in meta_schema:
+        #         print(column_name)
+
+        # if len(meta_schema) == len(current_schema):
+        #     if set(meta_schema) != set(current_schema):
+        #         error_message = "Column in current file does not match " + \
+        #                 f"the meta_schema, got:\n {current_schema}. "
+        #         raise ValueError(error_message)
+        # else:
+        #     error_message = "Column in current file does not match " + \
+        #                 f"the meta_schema, got:\n {current_schema}. "
+        #     raise ValueError(error_message)
 
-        dataframe_list.append([file_name, current_df])
+        dataframe_list.append(current_df)
+        dataframe_name.append(file_name)
 
     print("CSVs are converted into dataframes and combined into a list!")
     print("Total of " + str(len(dataframe_list)) + " dataframes in the list.")
-    for each_file in dataframe_list:
-        print("File name: ", each_file[0])
+    for i, each_file in enumerate(dataframe_list):
+        print("File name: ", dataframe_name[0])

```

Comment:
@ruiheesi, we have being using the logging package across flag to have more flexibility. Would you please use logging.info  instead of print? Here is an example: https://github.com/FNLCR-DMAP/SCSAWorkflow/blob/3349efe4e5ff976a305692d531611ec4e5844aae/src/spac/data_utils.py#L644